package.json

```diff
@@ -94,8 +94,8 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^29.5.0",
     "prettier": "^2.8.8",
-    "react": "^18.2.0",
```

Comment:
having react 18 give this console warning
![Screenshot 2023-05-31 at 11 28 59 PM](https://github.com/thoughtback/rich-markdown-editor/assets/8145951/4767d021-7160-4910-8f29-4eb181eef5a5)